Studio-Lovelies/GG-JointJustice-Unity
# Allow setting of poses of off-screen actors

## Summary
Fixed bug where poses can't be set if an actor is off-screen and caused a warning to occur.

## Before/after screenshots and/or animated gif
N/A

## Testing instructions
1. Checkout this branch
2. Run the `ActorControllerTests::SetPoseOnInactiveActor()` test
3. Verify that the test passes
4. Revert the commit containing the fix (bd9c6c6bde03f495fe3281320c0ef994b7bb4693) and verify that the test fails

## Additional information
- `[x]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource:
  - closes #233
